tools/log-collector/helper.go

```diff
@@ -34,20 +34,20 @@ const (
 	BatchGv       = "batch/v1"
 	BatchGv1beta1 = "batch/v1beta1"
 	AppsGv        = "apps/v1"
-	extension     = "extensions/v1beta1"
+	networkingGv  = "networking.k8s.io/v1beta1"
 
 	Namespaces          = "namespaces"
 	Events              = "events"
 	CRD                 = "customresourcedefinitions"
 	StorageClass        = "storageclasses"
 	VolumeSnapshot      = "volumesnapshots"
 	VolumeSnapshotClass = "volumesnapshotclasses"
-	ConversionNamespace = "trilio-conversion"
```

Comment:
is this not needed? are we not collecting resources from `ConversionNamespace`?

Reply:
Conversion Namespace is now deprecated